pkgs/by-name/te/teams-for-linux/package.nix

```
@@ -0,0 +1,128 @@
+{
+  lib,
+  stdenvNoCC,
```

No reason to use this over
```suggestion
  stdenv,
```

---

You're right, looks like buildNpmPackage already uses stdenv.